File: seed/php-sdk/exhaustive/no-custom-config/src/EndpointsParams/Requests/EndpointsParamsGetWithAllowMultipleQueryRequest.php
<?php

namespace Seed\EndpointsParams\Requests;

use Seed\Core\Json\JsonSerializableType;

class EndpointsParamsGetWithAllowMultipleQueryRequest extends JsonSerializableType
{
    /**
     * @var ?array<string> $query
     */
    public ?array $query;

    /**
     * @var ?array<int> $number
     */
    public ?array $number;

    /**
     * @param array{
     *   query?: ?array<string>,
     *   number?: ?array<int>,
     * } $values
     */
    public function __construct(
        array $values = [],
    ) {
        $this->query = $values['query'] ?? null;
        $this->number = $values['number'] ?? null;
    }
}

File: seed/ruby-sdk-v2/exhaustive/wire-tests/lib/seed/endpoints_params/types/endpoints_params_get_with_allow_multiple_query_request.rb
# frozen_string_literal: true

module Seed
  module EndpointsParams
    module Types
      class EndpointsParamsGetWithAllowMultipleQueryRequest < Internal::Types::Model
        field :query, -> { String }, optional: true, nullable: false
        field :number, -> { Integer }, optional: true, nullable: false
      end
    end
  end
end
